source/developer/fx-guidelines.rst

```diff
@@ -86,15 +85,14 @@ Desktop
 
 -----
 
-
 UX Feedback Guidelines
 ****************************
 
-User experience design is a discipline drawing from ergonomics, ethnography, human factors research, and cognitive psychology. It's evaluated qualitatively with tools like heuristic evaluation and cognitive walkthroughs, and quantitatively through telemetry, anonymous usage data and other automated diagnostics.
+User experience design is a discipline drawing from ergonomics, ethnography, human factors research and cognitive psychology. It's evaluated qualitatively with tools like heuristic evaluation and cognitive walkthroughs, and quantitatively through telemetry, anonymous usage data and other automated diagnostics.
 
 As with any design process, iteration, prototyping, informed critique and evaluation is vital to doing our best work. Developing the skill of informed critique in our community increases the quality of our designs and of our product.
 
-This guide outlines the levels of UX feedback in order of positive impact in order to guide contributors towards the behaviors that best  advances a fast-moving, asychronously-developed open source product.
+This guide outlines the levels of UX feedback in the order of positive impact in order to guide contributors towards the behaviors that best  advances a fast-moving, asychronously-developed open source product.
```

Comment:
"towards the behaviors that best advances..." > "towards the behaviors that best advance..."?